preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -405,18 +405,34 @@ def identity(
         if not input_datum:
             return ProcessingResult(datum=None, warnings=[], errors=[])
 
+        warnings: list[ProcessingAnnotation] = []
         output_datum: ProcessedMetadataValue
         if args and "type" in args:
             match args["type"]:
                 case "int":
                     output_datum = int(input_datum)
                 case "float":
                     output_datum = float(input_datum)
+                case "boolean":
+                    if input_datum.lower() == "true":
+                        output_datum = True
+                    elif input_datum.lower() == "false":
+                        output_datum = False
+                    else:
```

Comment:
I would potentially not default to False but default to None (unless the field is required it should be ok it set it to None)

Reply:
SILO anyway always accepts `null`, so I think it's a good idea. I'll change it.